package.json

```diff
@@ -39,6 +39,8 @@
     "start": "rollup -c --watch",
     "build": "npm run lint && npm run rollup",
     "local-deploy": "npm run build && cp dist/maxi-media-player.js ~/homeassistant/www/community/maxi-media-player/maxi-media-player.js",
+    "local-deploythlhacs": "npm run build && xcopy \"dist\\maxi-media-player.js\" \"\\\\homeassistantvm\\config\\www\\community\\maxi-media-player\\maxi-media-player.js\" /Y /-I",
```

Comment:
These are to personal/specific to your home assistant setup. Can you remove them?